src/main/java/at/hannibal2/skyhanni/features/skillprogress/SkillUtil.kt

```diff
@@ -0,0 +1,142 @@
+package at.hannibal2.skyhanni.features.skillprogress
+
+import at.hannibal2.skyhanni.api.SkillAPI
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import com.google.gson.JsonArray
+import io.github.moulberry.notenoughupdates.util.Constants
+import io.github.moulberry.notenoughupdates.util.Utils
+
+object SkillUtil {
+
+    var exactLevelingMap = mapOf<Int, Int>()
```

Comment:
Please move the members and base logic in SkillAPI and just keep the math or utils methods in this class.